src/thor/map_matcher.cc

```diff
@@ -155,14 +155,19 @@ MapMatcher::FormPath(meili::MapMatcher* matcher,
       // get the timezone
       nodeinfo = tile->node(directededge->endnode());
       const auto* tz = DateTime::get_tz_db().from_index(nodeinfo->timezone());
+      if (!tz)
+        continue;
```

Comment:
its possible the data doesnt have timezone information, in which case we bail

Reply:
The a-star ignores missing timezone and continues anyway (but logs it). It would be nice if the general behaviour for missing timezones was the same everywhere, to avoid _yes but no_ answers.